frontend/src/pages/Home.js

```diff
@@ -1,23 +1,18 @@
 import React from "react";
-import Banner from "../components/shared/Banner/Banner";
-import Footer from "../components/shared/Footer/Footer";
-import MoviesList from "../components/Movies/MoviesList/MoviesList";
+import Banner from "../components/shared/Banner";
+import Footer from "../components/shared/Footer";
+import MoviesList from "../components/Movies/MoviesList";
 
 const Home = ({
-  deleteModal,
-  setDeleteModal,
   movieModal,
```

Comment:
Please, clean the props, there are still props that you're not using!!!!!

Reply:
these 2 props arent even in my Home.js component